src/components/thumbnail-carousel.ts

```diff
@@ -109,25 +109,50 @@ export class FrigateCardThumbnailCarousel extends FrigateCardCarousel {
       return;
     }
 
-    return html`<div
-      class="embla__slide"
-      @click=${(ev) => {
-        if (this._carousel && this._carousel.clickAllowed()) {
-          dispatchFrigateCardEvent<ThumbnailCarouselTap>(this, 'carousel:tap', {
-            slideIndex: slideIndex,
-            target: parent,
-            childIndex: childIndex,
-          });
-        }
-        stopEventFromActivatingCardWideActions(ev);
-      }}
-    >
-      <img
-        aria-label="${mediaToRender.title}"
-        src="${mediaToRender.thumbnail}"
-        title="${mediaToRender.title}"
-      />
-    </div>`;
+    if (mediaToRender.frigate?.event?.retain_indefinitely == true) {
+      return html`<div
+        class="embla__slide"
+        @click=${(ev) => {
+          if (this._carousel && this._carousel.clickAllowed()) {
+            dispatchFrigateCardEvent<ThumbnailCarouselTap>(this, 'carousel:tap', {
+              slideIndex: slideIndex,
+              target: parent,
+              childIndex: childIndex,
+            });
+          }
+          stopEventFromActivatingCardWideActions(ev);
+        }}
+      >
+        <img
+          aria-label="${mediaToRender.title}"
+          src="${mediaToRender.thumbnail}"
+          title="${mediaToRender.title}"
+        />
+        <img
+          class="favorite"
+        />
```

Comment:
No `src`? I was assuming you'd use a `<ha-icon>` here with `mdi:heart` or equivalent to show it being a favorite?

Reply:
Yeah this is another thing that I wasn't sure on how to do so I was just trying to get something to see if it showed up. That should be really easy to do then 👍
Probably will do ⭐  since that is what frigate uses after Blake's UI redo

Reply:
Star works. Here's an example from elsewhere of using the `ha-icon` component: https://github.com/dermotduffy/frigate-hass-card/blob/main/src/components/menu.ts#L252